changelog/issue-2876.md

```diff
@@ -0,0 +1,5 @@
+audience: general
+level: patch
+reference: issue 2876
+---
+<!-- replace this text with your changelog entry.  See dev-docs/best-practices/changelog.md for help writing changelog entries. -->
```

Comment:
```suggestion
The purge cache UI view now allows filtering a search result by cache name.
```

Reply:
I missed this. From now on I will remember to add description to changelog. Thanks :)